docs/csharp/tutorials/console-teleprompter.md

```diff
@@ -128,22 +129,22 @@ indicates that there are no more items.
 
 There are two other C# syntax elements that may be new to you. The `using` 
 statement in this method manages resource cleanup. The variable that is
-initialized in the using statement (`reader`, in this example) must
+initialized in the `using` statement (`reader`, in this example) must
 implement the `IDisposable` interface. The `IDisposable` interface
-defines a single method, `Dispose()`, that should be called when the
+defines a single method, `Dispose`, that should be called when the
```

Comment:
Here again, it might be better to change these from `IDisposable` to @System.IDisposable, and from `Dispose` to @System.IDisposable.Dispose so that they resolve as links.